features/facebookImport/src/model/importers/multiple-files-importer.js

```diff
@@ -41,51 +40,48 @@ export default class MultipleFilesImporter extends Importer {
             });
     }
 
-    async _importDataFromFiles(targetEntries, zipFile, facebookAccount) {
-        const fileDataWithResults = await Promise.all(
-            targetEntries.map((targetEntry) =>
-                this._readJSONFileWithStatus(targetEntry, zipFile)
-            )
-        );
-
-        const successfullResults = fileDataWithResults.filter(
-            (result) => result.status.isSuccess
-        );
-
-        for (const each of successfullResults) {
-            facebookAccount.addImportedFileName(each.targetEntry.path);
-        }
-        const dataResults = successfullResults.map((result) => result.data);
-        this._importRawDataResults(facebookAccount, dataResults);
-
-        return fileDataWithResults
-            .filter((result) => !result.status.isSuccess)
-            .map(({ status }) => status);
-    }
-
     _createMissingFilesError() {
         return new MissingFilesException();
     }
 
-    async import({ zipFile, facebookAccount }) {
+    async import({ zipFile }) {
```

Comment:
There's either a naming nit or a fundamental part of the architecture I'm missing. Main question is, where's the pod?
Before, we were dealing with the pod through the account. But now, we're dealing with zip files directly here?
Additionally, we're assuming that zipFile has a `getEntries`; the problem is that does not work all over. And now that I see the code path, it's not a problem with this specific PR, so it can probably be safely ignored.
Let's just keep it in the back of our mind, as usual.